magenta/music/melody_encoder_decoder.py

```diff
@@ -0,0 +1,428 @@
+
```

Comment:
I was going to remove it since it doesn't seem to be any better than lookback, but I just added it back in for backwards compatibility.